packages/react-dom/src/events/DOMModernPluginEventSystem.js

```diff
@@ -619,7 +632,7 @@ function createDispatchQueueItem(
   };
 }
 
-export function accumulateTwoPhaseListeners(
+export function accumulatePhaseListeners(
```

Comment:
Any particular reason for rename? Is it not always two phases?

Reply:
Reverted this change.